src/scenes/Projects/Overview/ProjectOverview.graphql

```diff
@@ -0,0 +1,58 @@
+query ProjectOverview($input: ID!) {
+  project(id: $input) {
+    id
+    name {
+      value
+    }
+    location {
+      value {
+        ...DisplayPlace
+        region {
+          value {
+            name {
+              value
+            }
+          }
+        }
```

Comment:
```suggestion
        ...DisplayLocation
```

Reply:
Uses DisplayLocation instead of DisplayPlace